torch/csrc/jit/ir.h

```diff
@@ -136,6 +199,9 @@ struct Node : public Attributes<Node> {
   bool hasType() const {
     return type_ != nullptr;
   }
+  virtual std::string name() const {
```

Comment:
Do we need to add this method? The problem with names is that we already have `uniqueName()` and `debugName()`, and `name()` sounds like sth even more important, but really it returns an empty string for most of our nodes

Reply:
We don't, removing.